src/lib/viewers/text/PlainTextViewer.js

```diff
@@ -113,8 +113,8 @@ class PlainTextViewer extends TextBaseViewer {
         super.setup();
 
         this.textEl = this.createViewer(document.createElement('pre'));
-        this.textEl.className = 'bp-text bp-text-plain hljs';
-        this.textEl.classList.add(CLASS_HIDDEN);
+        this.textEl.className = `bp-text bp-text-plain hljs ${CLASS_BOX_PREVIEW_CONTENT_BODY} ${CLASS_HIDDEN}`;
```

Comment:
Could the setting of tabIndex & classes be something we extend the `createViewer` method with?

Reply:
tabIndex probably not since we don't want certain viewer's content div to be focusable. I'll see how big the change would be for classes